databricks_cli/jobs/cli.py

```diff
@@ -111,19 +115,20 @@ def list_cli(output):
         click.echo(tabulate(_jobs_to_table(jobs_json), tablefmt='plain'))
 
 
-@click.command(context_settings=CONTEXT_SETTINGS)
-@click.option('--job-id', required=True, type=int)
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help="Delete the specified job from the Databricks Job Service.")
```

Comment:
* `Deletes`
* Remove `from the Databricks Job Service`. The context is sufficient.